.travis.yml

```diff
@@ -32,6 +32,11 @@ cache:
 # If inline scripts get too long, Travis surprisingly prints them twice.
 
 before_install:
+  # Travis pre-installs an old version of numpy. Uninstalling it
+  # manually has been known to solve some strange failures that we've
+  # had in the past.
+  - pip uninstall -y numpy
+  - pip freeze
```

Comment:
Can you add a comment that this is (presumably) for debugging?

Reply:
Sure. (It is for debugging, and I kept it in intentionally.)